rosbag2_transport/include/rosbag2_transport/play_options.hpp

```diff
@@ -16,6 +16,7 @@
 #define ROSBAG2_TRANSPORT__PLAY_OPTIONS_HPP_
 
 #include <cstddef>
+#include <cstdint>
```

Comment:
```suggestion
```

Reply:
Addressed in https://github.com/ros2/rosbag2/pull/931/commits/07543e0c770db06fbada46268105fd4eea1a9d3a.